test/diagnostics/incomplete-signature-doc.lua

```diff
@@ -68,6 +73,12 @@ function FGPP2(p0, <!p1!>)
   print(p0, p1)
 end
 
+---comment
+---@param p1 any
+function FGPP2_(<!p0!>, p1)
```

Comment:
I'd suggest that we do not mix test case where we expect a warning with those where we do not.
And in order to clarify or expectations I'd always put a comment above `TEST [[`, sort of like the test method name in other test frameworks.